go/cmd/pfacct/radius.go

```diff
@@ -134,25 +143,34 @@ func (h *PfAcct) handleAccountingRequest(rr radiusRequest) {
 	timestamp = timestamp.Truncate(h.TimeDuration)
 	node_id := mac.NodeId(uint16(switchInfo.TenantId))
 	unique_session_id := h.accountingUniqueSessionId(r)
-	if err := h.InsertBandwidthAccounting(
-		node_id,
-		switchInfo.TenantId,
-		mac.String(),
-		unique_session_id,
-		timestamp,
-		in_bytes,
-		out_bytes,
-	); err != nil {
-		logError(ctx, "InsertBandwidthAccounting: "+err.Error())
-	}
-
-	if status == rfc2866.AcctStatusType_Value_Stop {
-		h.CloseSession(node_id, unique_session_id)
-	}
+	node_acct_info := acct_info{node_id: node_id,
+		tenant_id:      switchInfo.TenantId,
+		mac:            mac.String(),
+		unique_session: unique_session_id,
+		bucket:         timestamp,
+		in_bytes:       in_bytes,
+		out_bytes:      out_bytes,
+	}
+	go func() {
```

Comment:
Now that the async is handled by the JobChan, do we really need to wrap this into a 'go func' ?

Reply:
The JobChan is to queue the sql requests but if the api call stuck i still want to reply to the accounting packet right away.